### PR TITLE
Improve documentation

### LINKED ISSUE #163: [documentation] add model assumptions
- [x] most important assumptions of optimization/simulation (MVS)
- [x] area potential
- [x] maximumCap pv
- [x] reference to MVS

### PR DESCRIPTION
**Pull request by SabineHaas**
Fix #163,  #159

**Changes proposed in this pull request**:
- Introduce section "local energy system" in Model assumptions in RTD
-  Add basic model assumptions to RTD
- Add headings in `parameters.rst` to make references of these sections possible in RTD
- Improve "scope and limitations" section of RTD with additional information and corrections
- Minor changes in `README.rst`

The following steps were realized, as well (required):
- [x] Update the CHANGELOG.md
- :x: Check if full simulation tests pass locally (`EXECUTE_TESTS_ON=master pytest`)

Also the following steps were realized (if applies):
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [ ] For new functionalities: Explain in readthedocs
- [ ] Write test(s) for your new patch of code

*Please mark above checkboxes as following:*
- [ ] Open
- [x] Done

In case of an error due to linting, run `black . --exclude docs/` and push your changes.
Note that in case you do not fix a whole issue you should start your PR with `Address #xyz`.

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl-institut/mvs_eland/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by SabineHaas** [comment]
@MaGering I've included some more changes in the same PR so that it's easier to review;)

File: README.rst [hunk 1]
```diff
@@ -32,13 +32,14 @@ The functionalities include
 
 * calculation of an area potential for PV on roof-tops and facades based on building parameters,
```
Comment (MaGering) [inline README.rst:33]:
```suggestion
* calculation of an area potential for PV on roof-tops and façades based on building parameters,
```

File: docs/scope_limits.rst [hunk 2]
```diff
@@ -5,13 +5,24 @@ Scope and limitations
 
 The *pvcompare* model was developed during the H2020 project `GRECO <https://www.greco-project.eu/>`_ with the purpose of
```
Comment (MaGering) [inline docs/scope_limits.rst:6]:
```suggestion
The *pvcompare* model evolved during the H2020 project `GRECO <https://www.greco-project.eu/>`_ with the purpose of
```

Due to the duplication below.

File: docs/scope_limits.rst [hunk 3]
```diff
@@ -5,13 +5,24 @@ Scope and limitations
...
 - contentrator-PV (CPV),
 - a multi-junction of perovskite and silicon (PSI) and
-- PV powered heat pumps and chillers.
+- PV powered heat pumps.
 
-Therefore, the model focuses on Photovoltaiks, while also including sector-coupling through heat pumps and chillers.
-However, a shift of focus is possible and can easily integrated. Feel free to contribute and check out our `Contributing Guidelines <https://github.com/greco-project/pvcompare/blob/dev/CONTRIBUTING.md>`_.
+Therefore, the model focuses on photovoltaics, while including sector-coupling through heat pumps (and chillers).
+However, a shift of focus is possible and can easily be integrated.
+Concering the components of the energy system you can use all components listed in the `MVS documentation <https://multi-vector-simulator.readthedocs.io/en/v0.5.5/Model_Assumptions.html#component-models>`_.
```
Comment (MaGering) [inline docs/scope_limits.rst:16]:
```suggestion
Concerning the components of the energy system, you can use any of the components listed in the `MVS documentation <https://multi-vector-simulator.readthedocs.io/en/v0.5.5/Model_Assumptions.html#component-models>`_.
```

File: docs/scope_limits.rst [hunk 4]
```diff
@@ -5,13 +5,24 @@ Scope and limitations
...
+In this link you also find information on packages for calculating feed-in time series of other fluctuating renewable energies than PV.
```
Comment (MaGering) [inline docs/scope_limits.rst:17]:
```suggestion
There you also find information on packages for calculating feed-in time series of other fluctuating renewable energies than PV.
```

File: docs/scope_limits.rst [hunk 5]
```diff
@@ -5,13 +5,24 @@ Scope and limitations
...
+Checkout :ref:`define_params` for information on how to include more components into the simulated energy system.
 
-More information on the scope and limitations will follow.
\ No newline at end of file
+*pvcompare* further focusses on residential areas in cities (also see :ref:`local_energy_system`).
+Therefore, assumptions for demand and buildings (available area for PV on façades) are based on this use case.
+However, users can provide their own demand time series, adjust the building assumptions (see :ref:`building_parameters`) or only allow roof-top PV (see :ref:`pv_setup`) in their simulations.
+This way other use cases like detached houses (single-family houses) or industrial areas can be analyzed, as well.
+
+The energy system modelled with *pvcompare* has only one node, meaning that within the energy system energy can be shared without restrictions through power connections between the buildings ("Coper Plate").
```
Comment (MaGering) [inline docs/scope_limits.rst:25]:
```suggestion
The energy system modelled with *pvcompare* has only one node, meaning that within the energy system, energy can be shared without restrictions through power connections between the buildings ("Coper Plate").
```

File: docs/scope_limits.rst [hunk 6]
```diff
@@ -5,13 +5,24 @@ Scope and limitations
...
+An analysis of the interconnection of several buildings with restrictions in power lines or due to billing are not content of this model.
+
+As *pvcompare* builds on other packages there are inherited limitations. Checkout the `limitations of the MVS <https://multi-vector-simulator.readthedocs.io/en/v0.5.5/Model_Assumptions.html#limitations>`_.
```
Comment (MaGering) [inline docs/scope_limits.rst:28]:
```suggestion
As *pvcompare* builds on other packages, there are inherited limitations. Checkout the `limitations of the MVS <https://multi-vector-simulator.readthedocs.io/en/v0.5.5/Model_Assumptions.html#limitations>`_.
```

**Review by MaGering — APPROVED** [review]
Thank you for improving the docs! :) There are a few typos I found during the review. The content looks good to me!

A typo in README.rst in line 74 / 75 that I couldn't correct in review mode. It should be: 

```
You can download the open source [...]
```

File: README.rst [hunk 7]
```diff
@@ -32,13 +32,14 @@ The functionalities include
...
 * calculation of heat and electricity demand profiles for a specific amount of people living in these buildings,
-* calculation of PV feed-in time series for a set of PV installations on roof-tops and facades incl. different technologies,
+* calculation of PV feed-in time series for a set of PV installations on roof-tops and façades incl. different technologies,
 
     * all technologies in the database of `pvlib <https://pvlib-python.readthedocs.io/en/stable/index.html>`_,
-    * a specific concentrator-PV module, and
-    * a module of silicon-perovskite cells,
+    * a specific concentrator-PV module (see :ref:`cpv`) and
+    * a module of silicon-perovskite cells (see :ref:`psi`),
```
[the remark this answers is not in the document]
Comment (SabineHaas) [reply README.rst]: ahh, thank! done with 3272c65

**Comment by SabineHaas** [comment]
> Thank you for improving the docs! :) There are a few typos I found during the review. The content looks good to me!
> 

Thank you @MaGering for correcting the typos and some expressions :)

> A typo in README.rst in line 74 / 75 that I couldn't correct in review mode. It should be:
> 
> ```
> You can download the open source [...]
> ```

done with bdafd38

**Pull request merged**